R/method-correlation.R

```diff
@@ -298,12 +325,12 @@ setMethod("selfCorrelation", signature("SampleMetadata"), function(data, method
 #' This is essentially an alias to the microbiomeComputations::correlation function.
 #' 
 #' @param data a data.table
-#' @param method string defining the type of correlation to run. The currently supported values are 'spearman' and 'pearson'
+#' @param method string defining the type of correlation to run. The currently supported values are 'spearman', 'pearson' and 'sparcc'
 #' @param verbose boolean indicating if timed logging is desired
 #' @return ComputeResult object
 #' @import veupathUtils
 #' @export
-setMethod("selfCorrelation", signature("data.table"), function(data, method = c('spearman','pearson'), verbose = c(TRUE, FALSE)) {
+setMethod("selfCorrelation", signature("data.table"), function(data, method = c('spearman','pearson','sparcc'), verbose = c(TRUE, FALSE)) {
   correlation(data, method=method, verbose=verbose)
 })
 
```

Comment:
Why not sparcc as a method option in line 343? We could be correlating two compositional assays there

Reply:
yea ive thought about it a fair bit actually.. im not sure the compositional adjustment is appropriate if youre correlating across two different compositions.